src/core/electrostatics_magnetostatics/coulomb.cpp

```diff
@@ -0,0 +1,555 @@
+
+#include "coulomb.hpp"
+#include "communication.hpp"                         // bcast functions
+#include "electrostatics_magnetostatics/p3m_gpu.hpp" // p3m_gpu_init
+#include "errorhandling.hpp"                         // runtime_error
+#include "initialize.hpp"                            // on_ghost_flags_change
+#include "layered.hpp"                               // layer_h
+
+#ifdef ELECTROSTATICS
+
+namespace Coulomb {
+
+void pressure_n_coulomb(int &n_coulomb) {
```

Comment:
Can you please remove all the `coulomb`/`dipolar` parts from the names that are redundant with the namespaces?

Reply:
You can also remove the prefixed like `pressue_` and `nonbonded_`, it should be clear by the name of the function what it doesn.